src/amp-ads/40_Experimental_Ads/Carousel_Lightbox_Ad.html

```diff
@@ -0,0 +1,418 @@
+        border-top: 1px solid #eee;
+        padding: 18px 0;
+      }
+      .lightbox-header {
+        background: #000;
+        text-align: center;
+        padding: 10px 0;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .lightbox-header h1  {
+        display: inline;
+        color: #ffffff;
+        text-transform: uppercase;
+        font-weight: normal;
+        font-size: inherit;
+        border-left: 1px solid #ffffff;
+        margin: 0 0 0 12px;
+        padding-left: 12px;
+      }
+      .lightbox-hero {
+        position: relative;
+        overflow: hidden;
+      }
+      .learn-more a {
+        padding: 8px 0;
+        text-transform: uppercase;
+        display: inline-block;
+        color: #2979ff;
+        text-decoration: none;
+        font-weight: bold;
+      }
+      /* Responsive styles so that the lightbox does not over-stretch */
+      @media (max-height: 500px) {
+        .lightbox {
+          max-width: 400px;
+        }
+        .lightbox-hero {
+          max-height: 160px;
+        }
+        .lightbox-hero-img {
+          margin-top: -30px;
+        }
+      }
+      @media (max-height: 360px) {
+        .lightbox-hero-img {
+          display: none;
+        }
+        .lightbox-header {
+          position: static;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <!-- ## Banner markup -->
+    <!--
+    `amp-ad-banner` is a dummy AMP element that is used by the AMP runtime to determine the position of the ad when a lightbox is opened.
+
+    Please note that `amp-ad-banner` is still in experimental stages and is likely to change in the future. See [this Github issue for tracking.](https://github.com/ampproject/amphtml/issues/8334)
+    -->
+    <amp-ad-banner class="root-container">
+      <div class="main-container">
+        <amp-img src="<%host%>/img/car-side-bg.jpg"
+            width="608"
+            height="342"
+            layout="responsive"
+            class="bg-img"></amp-img>
+        <!-- ## Carousel -->
+        <!--
+        We use the `amp-carousel` to display multiple pieces of content along a horizontal axis.
+
+        All elements with the `carousel-item` class-name have essentially the same markup: one container that activates the lightbox, an image, and a button.
+
+        AMP has a system in place for [events and actions](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md). It uses a domain-specific language to describe how actions are triggered. In this example, we set the `on` attribute so the lightbox will activate when a carousel item is tapped.
+        -->
+        <amp-carousel
+            height="132"
+            layout="fixed-height"
+            type="carousel"
+            class="carousel">
+          <div class="carousel-item"
+              on="tap:main-lightbox.activate"
```

Comment:
I think you can move this to the carousel element and remove it from all carousel items.